src/stream.c

```diff
@@ -502,10 +524,10 @@ int stream_alloc(struct stream **sp, struct list *streaml,
 		goto out;
 
 	/* Jitter buffer */
-	if (cfg->jbuf_del.min && cfg->jbuf_del.max) {
+	if (jbuf && cfg->jbuf_del.min && cfg->jbuf_del.max) {
 
 		err = jbuf_alloc(&s->jbuf, cfg->jbuf_del.min,
-				 cfg->jbuf_del.max);
+				 cfg->jbuf_del.max, cfg->jbuf_wish, ptime);
```

Comment:
is there a need to send `ptime` to jbuf ?

Reply:
Yes, the adaptive jitter buffer algorithm needs ptime. See libre PR! Earlier I had a version where ptime was somehow computed from the RTP timestamps. But it was not very reliable.